jetstream/cli.py

```diff
@@ -95,16 +97,24 @@ def execute(
             {"date": date.strftime("%Y-%m-%d"), "slug": slug} for (slug, date) in worklist
         ]
 
+        worklist_groups = groupby(worklist, lambda x: x[0])
+        experiment_enrollments = [
+            {"slug": slug, "date": max(group, key=lambda x: x[1])[1]}
+            for slug, group in worklist_groups
+        ]
+
```

Comment:
This part is mostly relevant when re-running an analysis. The enrollments table needs to be created only once. When creating the enrollments table the date of the last experiment run should be considered, hence the `max(...)`.